### PR TITLE
Allow admins to edit any project

### PR DESCRIPTION
**Pull request by gr-eg**
Add two new routes under the admin namespace:
```
/admin/events/cp-2018/projects/:id/edit
/admin/events/cp-2018/projects/:id/extra
```
These render new components with a warning message to the admin that they are editing a live project and then the existing component.

### CODE REVIEW

File: client/src/admin/projects/ExtraDetails.vue [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<template>
+  <div v-if="event.id && project.id">
+    <div class="center-grid">
+      <div class="center-grid__content">
+        <div class="error-message-warning">
+          You are an admin editing a live project
+        </div>
+        <extra-details :eventSlug='this.event.slug' :projectId='project.id' :_event='this.event' :_project='project'></extra-details>
```
Comment (Wardormeur) [inline client/src/admin/projects/ExtraDetails.vue]: i don' t believe you need "this" in the template

Comment (gr-eg) [reply client/src/admin/projects/ExtraDetails.vue]: Good to know, thanks!

**Comment by conoramurphy** [comment]
Edit in a new tab is fine if that's easiest?

**Comment by gr-eg** [comment]
@DanielBrierton Good point thanks, I've just updated to redirect back to the admin list view page after editing a project if you are an admin.

**Pull request merged**